### DISCUSSION TITLE
How enable Trait & new PrintLabel

### DISCUSSION THREAD

**Discussion by Kantoborgy** [Symbiota/community-board#167] [Problems/Bugs]
Hello.

1. For enable PrintLabels just rename ../content/collections/reports/labeljson_template.php to labeljason.php & then chmod 777 to this file... ?
2. How enable Traits option & funcionality into omoccuredits

Thanks in advance

**Comment by arbolitoloco** [comment]
@kantobogy You're exactly right for the labels, but you don't have to chmod 777, chmod 755 is good enough (the server preferably should be the file owner)

**Reply by Kantoborgy** [reply]
@arbolitoloco Thanks 
And about enable Traits option..?
